code/modules/holodeck/computer.dm

```diff
@@ -72,7 +79,7 @@
 			linked.power_usage = new /list(AREA_USAGE_LEN)
 
 	generate_program_list()
-	load_program(offline_program, FALSE, FALSE)
+	load_program("holodeck_offline")//honestly there isnt a reason to do this as far as im aware
```

Comment:
until someone want custom offlike holodeck, like space, or plasma burn range or etc.

Reply:
fixed, now uses the offline_program var again in case someone wants to var edit a custom holodeck